### ISSUE TITLE
Accessibility issue: contrast issue with 'We would love to build your site' 

### ISSUE THREAD

**Issue by yvonne-liu** [FAC10/week1-Facalacalaca#64]
Warning reads: 

Text elements should have a reasonable contrast ratio (1)

See [AX_COLOR_01](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01) for more information.

`<figurecaption>​We would love to build your site​</figurecaption>​`

Try this link for contrast ratio check: http://webaim.org/resources/contrastchecker/
